RestaurantManagementSystem/RestaurantManagementSystem/src/Core/Entities/RegistrationRequest.cs

```diff
@@ -46,24 +46,26 @@ public class RegistrationRequest
         }
         //create method
         public static RegistrationRequest Create(string name, string email
-            , string address, ContactInfo contact, RequestStatus.Status status, string verificationcode
+            , string address, string contactinfoemail, string contactinfophonenumber,Status status, string verificationcode
             )
         {
             Guid id = Guid.NewGuid();
             DateTime datecreated = DateTime.Now.ToUniversalTime();
```

Comment:
dataCreated

Reply:
what's wrong ?

Reply:
if the variable name consists of two words you should use camelCase, so instead of datecreated it should be dateCreated.